plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ConfigureDependentFrameworksListener.java

```diff
@@ -240,7 +235,6 @@ public ADBInfo(HashMap<String,String> env) {
 				URI url = new URI(dockerHost);
 				String h = url.getHost();
 				openshiftHost = "https://" + h;
-				//openshiftPort = url.getPort();  // i don't believe this is the correct port. This is for docker
 			} catch(URISyntaxException murle) {
```

Comment:
Why should this be catched? Shouldn't this make it to the caller that then can react accordingly?